### ISSUE TITLE
Proxito: document custom prefixes feature

### ISSUE THREAD

**Issue by stsewd** [readthedocs/readthedocs.org#11324]
## What's the problem this feature will solve?

Custom prefixes for projects and subprojects were implemented in https://github.com/readthedocs/readthedocs.org/pull/10156, but we never documented them publicly. We have some internal docs about it in https://github.com/readthedocs/readthedocs.org/blob/main/readthedocs/proxito/README.rst#custom-path-prefixes.

We mention this feature in our pricing page (https://about.readthedocs.com/pricing/).

>  URL customization - Fine tune URLs used for your projects

We should link to our documentation from there with all the use cases this feature solves (proxying, removing or changing the "/projects" prefix).

There is also the discussion if we should expose this to users or manually handle this via support.

**Comment by agjohnson** [comment]
Yeah, I wouldn't mind exposing control of this as a beta feature/admin page as we are documenting it. Controlling this on behalf of users is tricky and requires scheduling changes and some back and forth. If users have this control they can remedy problems quicker.

**Comment by humitos** [comment]
We are writing a blog post to promote this feature in https://github.com/readthedocs/website/pull/356 but it seems we don't have documentation for the feature yet.

**Comment by stsewd** [comment]
That blog post isn't related to this feature, but to the ability of changing the version slug.

**Comment by humitos** [comment]
@ericholscher adding this to you since you have a PR already open and almost finished. It would be good to merge during this sprint.
